### ISSUE TITLE
As A Developer, I Want To Be Able To Sort By Label

### ISSUE THREAD

**Issue by justinbeal** [ZenHubIO/support#400]
Our organization uses github labels pretty freely to categorize and describe issues. Issues frequently have multiple labels associated with them (e.g. "high-importance", "frontend", "needs-design"), and we spend a lot of time juggling issues in swimlanes to sort by permutations of these labels. It would be nice to be able to specify a weight or score for each label and have ZenHub sort issues in the swimlanes based on the sum of the applicable labels' scores.

**Comment by Mottie** [comment]
I have a userscript that will allow you to sort a pipeline by assignee, time estimate, issue number, repo name and issue title: https://github.com/Mottie/ZenHub-userscripts

I'm not quite sure how one would sort by labels without an assigned value... any ideas?

![zenhub-sort-issues](https://cloud.githubusercontent.com/assets/136959/14061332/ec60109a-f34b-11e5-8ae8-118ab64a45a8.gif)

**Comment by robneville73** [comment]
+1...same here. In fact this would be a so-so compromise to horizontal swimlanes #304

**Comment by AndreasBackx** [comment]
Could this be taken a look at again? Would love to sort by labels. This would also solve #461 because you'd be able to sort on priority labels.

**Comment by mowies** [comment]
Yes please take a look at this again! My team uses labels as priorities and it would be very useful to sort our tickets by priority

**Comment by ConradSelig** [comment]
+1 I would love to have this.

Perhaps a more simple solution could be provided if all the described ideas is too much, have three default labels that are sorted.

"High" shows up first
"Medium" shows up in the middle
"Low" shows up at the bottom
Everything else shows up below that.

I would still love to be able to give my own labels priority levels, but this could be an easy first step.
